components/tensorflow-notebook-image/Dockerfile

```diff
@@ -113,7 +113,7 @@ RUN pip3 --no-cache-dir install \
     jupyter-console==6.0.0 \
     jupyterlab \
     xgboost \
-    kubeflow-fairing==0.7.1
```

Comment:
Can you upgrade to latest version 1.0.2?

Reply:
The dockerfile currently has 1.0.2 in it, so this change is causing a conflict.